modules/lss-missionHelper/missionHelper.user.js

```diff
@@ -733,7 +726,7 @@
     }
 
     var missionname = Normalize(document.getElementById("missionH1"));
```

Comment:
Lt. gängiger Namenskonvention werden Funktionen in JavaScript in lowerCamelCase geschrieben => normalize()